From 6fb8df5dcd886bbe2c6be082fbaf29b224f785b9 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:12:56 -0400
Subject: [ethernet] i40e/i40evf: Clean up code

Message-id: <1409663591-32531-134-git-send-email-sassmann@redhat.com>
Patchwork-id: 90608
O-Subject: [RHEL7.1 PATCH 133/148] i40e/i40evf: Clean up code
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Paul M Stillwell Jr <paul.m.stillwell.jr@intel.com>

1. Remove some break statements that will never get touched.
2. Remove an extra space.
3. Remove a comment for a parameter that doesn't exist
4. Move the assignment of a variable up to get rid of an else case.

Change-ID: I308a4b5ec070b1f0601f13b041ba4375aaad4b06
Signed-off-by: Paul M Stillwell Jr <paul.m.stillwell.jr@intel.com>
Tested-by: Jim Young <jamesx.m.young@intel.com>
Tested-by: Sibai Li <sibai.li@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit ec9a7db7f0438bc8ad69bc436cb3b3ed16642af9)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40e_adminq.c b/drivers/net/ethernet/intel/i40evf/i40e_adminq.c
index cc4b6db..8330744 100644
--- a/drivers/net/ethernet/intel/i40evf/i40e_adminq.c
+++ b/drivers/net/ethernet/intel/i40evf/i40e_adminq.c
@@ -817,7 +817,7 @@ i40e_status i40evf_asq_send_command(struct i40e_hw *hw,
    /* ugh! delay while spin_lock */
    udelay(delay_len);
    total_delay += delay_len;
-  } while (total_delay <  hw->aq.asq_cmd_timeout);
+  } while (total_delay < hw->aq.asq_cmd_timeout);
  }
 
  /* if ready, copy the desc back to temp */
-- 
1.7.1